carbon-mb/components/transports/mqtt/src/main/java/org/wso2/carbon/andes/transports/mqtt/adaptors/andes/utils/MqttUtils.java

```diff
@@ -300,4 +195,8 @@ public static boolean isDurable(boolean cleanSession, int qos) {
         return durable;
     }
 
+    public static ProtocolType getProtocolType() {
```

Comment:
Ideally this should not be the way of deciding on the protocol type of an MQTT message since this does not separate between different qos levels and protocol versions. Hence we should not write any new logic depending on this.